### DISCUSSION TITLE
ReferenceError: localStorage is not defined while ssr enabled on Nuxt

### DISCUSSION THREAD

**Discussion by yulafezmesi** [unicape/use-wagmi#163] [Q&A]
Hi! Possible to use `use-wagmi` with `ssr` enabled on nuxt without error? when i enabled SSR, giving above error.

![image](https://github.com/unicape/use-wagmi/assets/45617686/34f8d281-bed0-4c7a-a993-0fad884b2513)

**Comment by yulafezmesi** [comment]
just wanted to kindly tag you @unicape, in case you have got idea for this. thanks!

**Comment by brokenfiles** [comment]
Hi, any news?
